libs/ui-toolkit/pipes/src/dom-sanitizer.stub.ts

```diff
@@ -0,0 +1,4 @@
+export class DomSanitizerStub {
```

Comment:
we don't need to publish this stub, put it in pipes/testing or inside the only spec that use it (if we has only one)